### PR TITLE
Add cargo to dockerfile and build rustler

### PR DESCRIPTION
**Pull request by nelsonkopliku**
As per title adds cargo to the container and builds rustler.

@fabriziosestito remember me please what the issue was :sweat_smile:

### CODE REVIEW

**Comment by fabriziosestito** [comment]
Related to: https://github.com/philss/rustler_precompiled/pull/40

File: Dockerfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ ENV LANGUAGE en_US:en
 ENV LC_ALL en_US.UTF-8
 RUN zypper -n addrepo https://download.opensuse.org/repositories/devel:/languages:/erlang/SLE_15_SP3/devel:languages:erlang.repo
 RUN zypper -n --gpg-auto-import-keys ref -s
-RUN zypper -n in elixir
+RUN zypper -n in elixir cargo
```
Comment (fabriziosestito) [inline Dockerfile:7]: NITPICK: to be more explicit we are installing bot rust and cargo so: `rust cargo`

**Review by fabriziosestito — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (nelsonkopliku) [reply Dockerfile:7]: ok :+1:

File: Dockerfile [hunk 2]
```diff
@@ -18,7 +18,7 @@ COPY --from=elixir-build /build /build
 WORKDIR /build
 ARG MIX_ENV=prod
 ENV MIX_ENV=$MIX_ENV
-RUN mix phx.digest
+RUN RHAI_RUSTLER_FORCE_BUILD=true mix phx.digest
```
Comment (fabriziosestito) [inline Dockerfile:21]: note to self: we might want to consider making this part of application env in the future

**Pull request merged**